src/hotspot/share/opto/mulnode.cpp

```diff
@@ -433,6 +433,20 @@ const Type *MulFNode::mul_ring(const Type *t0, const Type *t1) const {
   return TypeF::make( t0->getf() * t1->getf() );
 }
 
+//------------------------------Ideal---------------------------------------
+// Check to see if we are multiplying by a constant 2 and convert to add, then try the regular MulNode::Ideal
+Node *MulFNode::Ideal(PhaseGVN *phase, bool can_reshape) {
```

Comment:
Please use the format `Type* identifier` for new code (in this case it is `PhaseGVN* phase`). The same applies to other places.

Reply:
Done